app/Models/Applicant.php

```diff
@@ -123,6 +130,13 @@ public function projects()
         return $this->morphMany(\App\Models\Project::class, 'projectable');
     }
 
+    public function classifications() //phpcs:ignore
+    {
+        return $this->belongsToMany(\App\Models\Classification::class, 'applicant_classification')
+            ->withPivot(['level', 'order'])
```

Comment:
I personally find the Laravel withPivot functionality a bit confusing. I think I'd prefer to create a new model class which represent the pivot table along with the new data. We've already done this with other models like ExperienceSkills (which could have been a pivot table with `->withPivot(['justification'])`. 
I think having the separate model makes it easier to reason about the data relationship.
I'm open to having my mind changed on this idea, but that's my current opinion. What do you think?

Reply:
I agree with you, thanks for clearing this up earlier!